web/server/watcher.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import json
+import typing as t
+from pathlib import Path
+
+from watchfiles import awatch
+
+from sqlmesh.core.context import Context
+from web.server.api.endpoints.models import get_models
+from web.server.settings import get_loaded_context, get_settings
+from web.server.sse import Event
+
+
+async def watch_project(queue: asyncio.Queue) -> None:
+    settings = get_settings()
+    context = await get_loaded_context(settings)
+
+    _get_path = _to_relative_path(context)
```

Comment:
You could've defined the function directly here and avoid another lambda:
```
def _get_path(path: str) -> str:
    return str(Path(path).relative_to(context.path))
```

Reply:
Is using lambda a bad pattern? I actually used it to avoid closure